plugins/nanospell/plugin.js

```diff
@@ -267,6 +254,9 @@
 				}
 				return true;
 			});
+			editor.on(EVENT_NAMES.START_RENDER, render, self);
+			editor.on(EVENT_NAMES.START_SCAN_WORDS, scanWords, self);
+			editor.on(EVENT_NAMES.START_CHECK_WORDS, checkWords, self);
```

Comment:
Just seemed to make sense to have all the event listener setup together.  I also like the pattern of making a `setupEventListeners` function to group them together and calling that from `init`